docs/metrics.md

```diff
@@ -134,6 +135,26 @@ In addition to those built-in metrics, the following metrics are added to the pi
 | rally.id |[uuid](https://mozilla.github.io/glean/book/user/metrics/uuid.html) |The id of the Rally client.  |[mozilla-rally/rally-core-addon#505](https://github.com/mozilla-rally/rally-core-addon/pull/505#issuecomment-815826426)||never | |
 | unenrollment.study_id |[string](https://mozilla.github.io/glean/book/user/metrics/string.html) |The id of the study user has left.  |[Bug 1646151](https://bugzilla.mozilla.org/show_bug.cgi?id=1646151#c32)||never | |
 
+## uninstall-deletion
+
+This ping is sent when user clicks the Join button for a study
+and accepts the study policy.
```

Comment:
This doesn't look like the correct conditions for the ping.

Reply:
Whoops, forgot to commit the updated docs.